vault/resource_quota_lease_count.go

```diff
@@ -0,0 +1,158 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/hashicorp/vault/api"
+)
+
+func quotaLeaseCountPath(name string) string {
+	return "sys/quotas/lease-count/" + name
+}
+
+func quotaLeaseCountResource() *schema.Resource {
+	return &schema.Resource{
+		Create: quotaLeaseCountCreate,
+		Read:   quotaLeaseCountRead,
+		// Update: quotaLeaseCountUpdate, // Due to hashicorp/vault#10712
+		Delete: quotaLeaseCountDelete,
+		Exists: quotaLeaseCountExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the quota.",
+				ForceNew:    true,
+			},
+			"path": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
It looks like the API allows this to be updated on an existing least count quota, so I think this can be false:

```suggestion
				ForceNew:    false,
```

Reply:
Updated